### PR TITLE
[build] create .hex from output + qorvo submodule bump

### PR DESCRIPTION
**Pull request by thcu-gp**
* Automatically create .hex files for qpg6095 and qpg6100 platform
* Link README.md to the correct documentation files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,17 +14,7 @@ This repo contains example platform drivers for [Qorvo GP712][gp712], [Qorvo QPG
...
-- [OpenThread on GP712 examples][gp712-page]
-- [OpenThread on QPG6095 examples][qpg6095-page]
-- [OpenThread on QPG6100 examples][qpg6100-page]
-- [OpenThread on QPG7015M examples][qpg7015m-page]
-
-[gp712-page]: ./src/gp712/README.md
-[qpg6095-page]: ./src/qpg6095/README.md
-[qpg6100-page]: ./src/qpg6100/README.md
-[qpg7015m-page]: ./src/qpg7015m/README.md
+To learn more about building and running the examples please check the [documentation](./third_party/Qorvo/repo/README.md)
```
Comment (jwhui) [inline README.md:17]: I don't think GitHub supports linking into a submodule in this way: https://github.com/openthread/ot-qorvo/blob/74763df6b2d0ccbe411d9ab69f95cb673044263e/third_party/Qorvo/repo/README.md

Comment (thcu-gp) [reply README.md]: replaced it with a link to the master branch

File: src/gp712/README.md [hunk 2]
```diff
@@ -1,63 +0,0 @@
-# OpenThread on Qorvo GP712 Example
```
Comment (jwhui) [inline src/gp712/README.md:1]: Also remove [src/qpg7015m/README.md](https://github.com/openthread/ot-qorvo/blob/main/src/qpg7015m/README.md)?

Comment (thcu-gp) [reply src/gp712/README.md:1]: indeed, removed it as well

**Pull request merged**